scss/page/header/_header.scss

```diff
@@ -0,0 +1,6 @@
+.header-logo{
+  text-align: center;
+  opacity: 0.2;
+  margin-top: 54px;
+  margin-left: 80px;
```

Comment:
marginy, paddingi mozesz zapisać w jednej linijce: margin: 54px 80px 0 0

działa to tak że to leci zgodnie ze wskazówkami zegara czyli pierwsze 0 to góra, drugie to lewo, i tak dalej

Reply:
Done